src/Magento/FunctionalTestingFramework/Test/etc/actionGroupSchema.xsd

```diff
@@ -30,6 +30,14 @@
                     </xs:sequence>
                 </xs:complexType>
             </xs:element>
+            <xs:element name="annotations">
```

Comment:
This would be backwards incompatible right?

Reply:
@aljcalandra Yes, mostly. We don't run schema validation by default on builds/etc, but if we changed that this would be breaking.
I'll create a ticket to make these required in the backlog for later implementation.